fvm/evm/stdlib/contract.cdc

```diff
@@ -0,0 +1,107 @@
+access(all)
```

Comment:
I would suggest not spending that much time reviewing stdlib part given in the next PR it has critically changed.